.travis.yml

```diff
@@ -56,27 +56,20 @@ env:
     - >-
       BUILD_TYPE=centaurHoricromtalEngineUpgradePapiV2alpha1
       BUILD_MARIADB=10.3
-    - >-
-      BUILD_TYPE=centaurPapiUpgradePapiV1
-      BUILD_MYSQL=5.7
-    - >-
-      BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV1
-      BUILD_MYSQL=5.7
-    - >-
-      BUILD_TYPE=centaurPapiUpgradePapiV2alpha1
-      BUILD_MYSQL=5.7
-    - >-
-      BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV2alpha1
-      BUILD_MYSQL=5.7
+    # This is not currently running any tests, see BT-81.
+    # - >-
+    #  BUILD_TYPE=centaurPapiUpgradePapiV2alpha1
+    #  BUILD_MYSQL=5.7
+    # This is not currently running any tests, see BT-81.
+    # - >-
+    #  BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV2alpha1
+    #  BUILD_MYSQL=5.7
```

Comment:
Revert, and use the same "syntax" as BA-6547: https://github.com/broadinstitute/cromwell/blob/54/src/ci/bin/testConformanceTesk.sh

Or, update testConformanceTesk.sh and .travis.yml to use this new format.

I just don't think jobs should be removed N different ways. I also prefer the other style of printing a message as it leaves a reminder of missing tests.